x/slashing/internal/types/querier.go

```diff
@@ -13,24 +13,12 @@ const (
 	QuerySigningInfos = "signingInfos"
 )
 
-// QuerySigningInfoParams defines the params for the following queries:
-// - 'custom/slashing/signingInfo'
-type QuerySigningInfoParams struct {
-	ConsAddress sdk.ConsAddress
-}
-
 // NewQuerySigningInfoParams creates a new QuerySigningInfoParams instance
 func NewQuerySigningInfoParams(consAddr sdk.ConsAddress) QuerySigningInfoParams {
 	return QuerySigningInfoParams{consAddr}
 }
 
-// QuerySigningInfosParams defines the params for the following queries:
-// - 'custom/slashing/signingInfos'
-type QuerySigningInfosParams struct {
```

Comment:
@alexanderbez should we be doing these as well? I didnt do it in the distr pr but can update in another PR if we are

Reply:
No, I didn't do these as well. There is no need to AFAIK. Revert please :)